src/System.Private.CoreLib/shared/System/Globalization/NumberFormatInfo.cs

```diff
@@ -86,6 +86,8 @@ sealed public class NumberFormatInfo : IFormatProvider, ICloneable
         // Is this NumberFormatInfo for invariant culture?
         internal bool m_isInvariant = false;
 
+        internal bool HasInvariantNumberSigns { get; private set; } = true;
```

Comment:
Nit: It may look better to follow the style in the rest of this type and use explicit `_hasInvariantNumberSigns` field here (feel free to change `m_isInvariant` to `_isInvariant` too to improve local consistency, but larger cleanups should be done separately).

Reply:
I ended up removing `m_isInvariant` entirely because it's never used anywhere.